test/opexecuter.cpp

```diff
@@ -116,8 +116,11 @@ void OpExecuter::ExecuteOnNGraph(vector<Tensor>& ngraph_outputs) {
   TF_CHECK_OK(BackendManager::GetCurrentlySetBackendName(&backend_name));
   tf::SessionOptions options = GetSessionOptions(backend_name);
   ClientSession session(tf_scope_, options);
-  TF_CHECK_OK(session.Run(sess_run_fetchoutputs_, &ngraph_outputs))
-      << "Failed to run opexecutor on NGTF";
+  try {
+    session.Run(sess_run_fetchoutputs_, &ngraph_outputs);
+  } catch (...) {
+    EXPECT_TRUE(false);
```

Comment:
Why is this needed?

Reply:
If exception happens while running a test in IE, our gtest execution is aborted. This fix allows to amrk that as failure and move on to try next tests in the suite.